teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```
@@ -423,7 +423,9 @@ private void initialise() {
             new RejectingSlashingProtector(),
             slashingProtectionLogger,
             new PublicKeyLoader(
-                externalSignerHttpClientFactory, validatorConfig.getValidatorExternalSignerUrl()),
+                externalSignerHttpClientFactory,
+                validatorConfig.getValidatorExternalSignerUrl(),
+                asyncRunner),
```

It does not make sense to have the retry behavior on the VoluntaryExitCommand. This is an interactive command and the user won't sit there waiting for this to eventually run... It should fail early in this case.

---

Also, when we fail to load the keys for VoluntaryExitCommand, the command just sits there and it looks like it is frozen (even though the retries are happening in the background, there is no feedback for the user). There is no output to the console.